lib/chewy.rb

```diff
@@ -1,6 +1,6 @@
 require 'active_support/version'
 require 'active_support/concern'
-require 'active_support/deprecation'
+require 'active_support/deprecation' if ActiveSupport.version < '7.1'
```

Comment:
You cannot compare like this, should be something 
```suggestion
require 'active_support/deprecation' if ActiveSupport.version < Gem::Version.new('7.1')
```

Reply:
That's completely true 🤦